src/test/java/com/group12/ciserver/CiServerApplicationTests.java

```diff
@@ -1,13 +1,56 @@
 package com.group12.ciserver;
 
+import com.group12.ciserver.database.DatabaseWrapper;
+import com.group12.ciserver.model.BuildInfo;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import java.util.Date;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
 @SpringBootTest
 class CiServerApplicationTests {
 
+	// use separate db for testing
+	private String test_database = "Testdatabase.db";
+
 	@Test
 	void contextLoads() {
 	}
 
+	@Test
+	void testAddBuild() {
+		DatabaseWrapper db = new DatabaseWrapper(test_database);
+		BuildInfo b = new BuildInfo(0, "commithash", "content", new Date());
+		long uid = db.addBuild(b);
+
+		assertTrue(db.getBuildInfo(uid) != null);
+	}
```

Comment:
I would recommend putting every test in this file in a separate test file, e.g. called `DatabaseWrapperTest.java`